packages/labs/eleventy-plugin-lit/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@lit-labs/eleventy-plugin-lit",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Eleventy plugin for rendering Lit components.",
+  "license": "BSD-3-Clause",
+  "homepage": "https://lit-element.polymer-project.org/",
+  "repository": "lit-html",
+  "main": "index.js",
+  "typings": "index.d.ts",
+  "exports": {
+    ".": {
+      "default": "./index.js"
+    }
+  },
+  "files": [
+    "/src/",
```

Comment:
Do we want `/src/` in files anymore? Our sourcemaps already include the original source.

Reply:
Removed.